src/api/index.ts

```diff
@@ -228,109 +226,113 @@ router.get(
     } catch (err) {
       res.status(500).json(err);
     }
-  },
+  }
 );
 
 router.post(
-  '/:installationId/sync',
+  "/:installationId/sync",
   bodyParser,
-  check('installationId').isInt(),
+  check("installationId").isInt(),
   returnOnValidationError,
   elapsedTimeMetrics,
   async (req: Request, res: Response): Promise<void> => {
     const githubInstallationId = Number(req.params.installationId);
     req.log.info(req.body);
-    const { jiraHost } = req.body;
+    const { jiraHost, resetType } = req.body;
 
     try {
       req.log.info(jiraHost, githubInstallationId);
       const subscription = await Subscription.getSingleInstallation(
         jiraHost,
-        githubInstallationId,
+        githubInstallationId
       );
 
       if (!subscription) {
         res.sendStatus(404);
         return;
       }
 
-      const type = req.body.resetType || null;
-      await Subscription.findOrStartSync(subscription, type);
+      await Subscription.findOrStartSync(subscription, resetType);
 
       res.status(202).json({
-        message: `Successfully (re)started sync for ${githubInstallationId}`,
+        message: `Successfully (re)started sync for ${githubInstallationId}`
       });
     } catch (err) {
       req.log.info(err);
       res.sendStatus(401);
     }
-  },
+  }
 );
 
-// Grab the last n failed syncs and trigger a sync
+// RESYNC ALL INSTANCES
 router.post(
-  '/resyncFailed',
+  "/resync",
```

Comment:
Would we need the original resyncFailed endpoint? (Should we consider `statusTypes` parameter instead?)

Reply:
yeah, don't need that one anymore as I've rolled it into the current one with sensible defaults.  If you wanted the same functionality, you can just set `statusTypes:['FAILED']` in the body.